tests/test_run_rvic.py

```diff
@@ -154,60 +133,30 @@ def test_run_full_rvic_online_valid(kwargs, client):
                 "grid_id": "COLUMBIA",
                 "run_startdate": "2012-12-01-00",
                 "stop_date": "2012-12-31",
-                "pour_points": url_path(
-                    "sample_pour.txt", "http", "climate_explorer_data_prep"
-                ),
-                "uh_box": resource_filename(
-                    "tests", "data/samples/uhboxes.csv"
-                ),  # Local file does not exist
-                "routing": url_path(
-                    "sample_flow_parameters.nc", "opendap", "climate_explorer_data_prep"
-                ),
-                "domain": resource_filename(
-                    "tests", "data/samples/sample_routing_domain.nc"
-                ),
-                "input_forcings": url_path(
-                    "columbia_vicset2.nc", "opendap", "climate_explorer_data_prep"
-                ),
-                "params_config_file": resource_filename(
-                    "tests", "data/configs/parameters.cfg"
-                ),
-                "params_config_dict": None,
-                "convolve_config_file": resource_filename(
-                    "tests", "data/configs/convolve.cfg"
-                ),
-                "convolve_config_dict": None,
+                "lons": "-118.0938",
+                "lats": "51.09375, 51.19375",  # Extra latitude
```

Comment:
Ah here. Well then maybe a test case that *correctly* uses multiple lats/lons :)